modules/mobile/app/controllers/mobile/v0/addresses_controller.rb

```diff
@@ -40,14 +40,6 @@ def validate
                          ))
         end
 
-        # No domestic or military addresses should have a province but some have been coming in as a string 'null'
```

Comment:
This is no longer happening. Pretty sure the bug is resolved.

Reply:
Yeah, this was happening very infrequently in production but we brought it to the upstream team and it was a small enough issue and required a big enough fix that it wasn't deemed worth doing. so no point in logging it anymore.